app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,10 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
```suggestion

```

aren't these no longer needed?

Reply:
@pJeyakumar, I configured Bootstrap and was working without it. We could face conflicts with your changes in the other commit; we will see.